src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements Area {
+    protected static final DecimalFormat df = new DecimalFormat("#.00");
```

Comment:
The use of a static formatter as a member of the abstract class may not be appropriate. If formatting is needed across multiple classes, consider using a utility class or method instead.